build.psm1

```diff
@@ -1634,7 +1634,7 @@ esac
     # OpenSUSE 42.1 (13.2 might build but is EOL)
     # Also SEE: https://fedoraproject.org/wiki/Packaging:DistTag
     if ($IsCentOS) {
-        $rpm_dist = "el7.centos"
+        $rpm_dist = "el7"
```

Comment:
Based on feedback about breaking existing users, we'll need to state the change so related scripts will be changed as part of https://github.com/PowerShell/PowerShell/issues/3888